src/tests/test_module.py

```diff
@@ -66,7 +66,8 @@ def testModuleInterface(self):
         self.assertEquals(type(System.__dict__), type({}))
         self.assertEquals(System.__name__, 'System')
         # the filename can be any module from the System namespace (eg System.Data.dll or System.dll)
-        self.assertTrue(fnmatch(System.__file__, "*System*.dll"))
+        self.assertTrue(fnmatch(System.__file__, "*System*.dll"),
+                        "unexpected System.__file__" + System.__file__)
```

Comment:
PS: I added the error message because I could not reproduce this on my machine and wanted to know what value it got in the integration tests.